kobo/apps/stripe/utils/misc.py

```diff
@@ -0,0 +1,72 @@
+from math import ceil, floor
+from typing import Optional
+
+from django.conf import settings
+
+from kobo.apps.organizations.models import OrganizationUser
+from kobo.apps.stripe.constants import ACTIVE_STRIPE_STATUSES
+from kobo.apps.stripe.utils.import_management import requires_stripe
+
+
+@requires_stripe
+def get_default_plan_name(**kwargs) -> Optional[str]:
+    Product = kwargs['product_model']
+    default_plan = (
+        Product.objects.filter(metadata__default_free_plan='true')
+        .values('name')
+        .first()
+    )
+    if default_plan is not None:
+        return default_plan['name']
+
+
+def generate_return_url(product_metadata):
```

Comment:
This and the method below are only called in views.py, maybe they should either move to that file or be their own view_utils.py file? I'm fundamentally opposed to having a 'misc' file